src/components/jellyfinActions.ts

```diff
@@ -239,59 +237,45 @@ export function play(state: PlaybackState): void {
  * get PlaybackInfo
  * @param item - item
  * @param maxBitrate - maxBitrate
- * @param deviceProfile - deviceProfile
  * @param startPosition - startPosition
  * @param mediaSourceId - mediaSourceId
  * @param audioStreamIndex - audioStreamIndex
  * @param subtitleStreamIndex - subtitleStreamIndex
  * @param liveStreamId - liveStreamId
  * @returns promise
  */
-export function getPlaybackInfo(
+export async function getPlaybackInfo(
     item: BaseItemDto,
     maxBitrate: number,
-    deviceProfile: DeviceProfile,
     startPosition: number,
     mediaSourceId: string,
     audioStreamIndex: number,
     subtitleStreamIndex: number,
     liveStreamId: string | null = null
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
 ): Promise<any> {
```

Comment:
```suggestion
): Promise<PlaybackInfoResponse> {
```

Could safely use the response type now here

Reply:
I would have but it has knock on effects at the call site so some refactoring would be needed. I wanted to avoid doing that in this PR but I can if it's preferred.

Reply:
Fair enough, let's keep it separate then